LowVoltage/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM centos:centos7
```

Comment:
adapt to naming scheme : LowVoltage directory -> low_voltage

Reply:
directory name changed: from LowVoltage directory to low_voltage